4_Methodology/BS_SD.adoc

```diff
@@ -292,6 +292,9 @@ The evaluator shall perform the following test to verify that the TOE generates
 The following test steps require the developer to provide access to a test platform that provides the evaluator with tools that are typically not found on factory products.
 
 . The evaluator shall perform biometric enrolment that results in creation of samples from which templates will be created that don’t satisfy the assessment criteria described in <<MBE assessment criteria for samples, assessment criteria for samples>> (e.g. presenting biometric samples of low quality)
++
+The evaluator may examine feedback messages (e.g. “Finger too wet” in case of fingerprint verification) shown when the TOE can’t capture the enough quality of samples to understand how the evaluator can present a low quality sample. The evaluator may also ask the developer how the low quality samples can be presented if the TOE doesn’t show any feedback messages.
```

Comment:
I may have steered you into using the term _quality_, but I believe this section should stick with not meeting the "assessment criteria." That likely includes low quality samples, but might also include samples exhibiting features of presentation attack.

Also suggest differentiating between sample (representation of biometric characteristic prior to feature extraction) and the characteristic itself.

```suggestion
. The evaluator may examine feedback messages shown (e.g. “Finger too wet” in case of fingerprint verification) to understand how the evaluator can present a biometric characteristic that does not meet the <<MBE assessment criteria for samples, assessment criteria for samples>>

. The evaluator may ask the developer how low quality samples can be created if the TOE doesn’t show any feedback messages
```